crates/smartmodule-development-kit/src/publish.rs

```diff
@@ -44,17 +45,19 @@ pub struct PublishCmd {
 impl PublishCmd {
     pub(crate) fn process(&self) -> Result<()> {
         let access = HubAccess::default_load(&self.remote)?;
-
         let opt = self.package.as_opt();
         let package_info = PackageInfo::from_options(&opt)?;
-
         let hubdir = package_info.package_relative_path(DEF_HUB_INIT_DIR);
-        if !hubdir.exists() {
-            init_package_template(&package_info)?;
-        } else if !self.public_yes {
-            check_package_meta_visiblity(&package_info)?;
+
+        if hubdir.exists() {
+            // Delete the `.hub` directory if already exists
+            tracing::warn!("Removing directory at {:?}", hubdir);
+            remove_dir_all(&hubdir)?;
```

Comment:
Given that we are removing directories under the hood, perhaps we could prompt the user before proceeding?
And allowing a flag like `-y` if we want to allow for CI use cases? WDYT @sehz , @digikata?

Reply:
dont think thats necessary.  user can back before publishing